docker/create_cluster.py

```diff
@@ -1,67 +1,62 @@
+from __future__ import print_function
+
 import argparse
 import subprocess
 import json
+import os
+import shutil
 
 import docker
 
 from keywords.exceptions import DockerError
 
 
-def remove_networks(docker_client):
+def remove_conflicting_network(docker_client, network_name):
     """ Removes all user defined docker networks """
 
-    # TODO: only remove network name that was passed
-
     networks = docker_client.networks.list()
 
-    # Filter out docker defined networks
-    predefined_networks = ['bridge', 'none', 'host']
-    user_defined_networks = [network for network in networks if network.name not in predefined_networks]
-
+    # If a network exists with the name of the one you requested to create
+    # remove the network and create a new one
```

Comment:
I would like to propose that this docker clean up not block this PR as I will be working on it right after in its own branch and there will be a lot of changes.